Makefile.common

```diff
@@ -32,14 +32,14 @@ H:= $(if $(VERBOSE),,@)  # not used yet
 TOP     = $(dir $(lastword $(MAKEFILE_LIST)))
 COQMAKE = $(MAKE) -f Makefile.coq $(COQMAKEOPTIONS)
 COQMAKE_TESTSUITE = $(MAKE) -f Makefile.test-suite.coq $(COQMAKEOPTIONS)
-BRANCH_coq:= $(shell $(coqtop) -v | head -1 | grep -E '(trunk|master)' \
+BRANCH_coq:= $(shell $(COQBIN)coqtop -v | head -1 | grep -E '(trunk|master)' \
```

Comment:
This doesn't work with Rocq 9.0 (without the coq shim). Let's rather keep `$(COQTOP)` and add (just below `COQBIN?=...` on top of the file something like `COQTOP?=$(shell command -v coqtop || echo "$(COQBIN)rocq repl")`